invoicer_project/static/js/registration_request.js

```diff
@@ -0,0 +1,330 @@
+const passwordMinLength = 8;
+const passwordMaxLength = 15;
+const nameSurnMaxLength = 35;
+const host = "http://127.0.0.1:8000";
+let csrf_token = document.getElementsByName('csrfmiddlewaretoken')[0].value;
```

Comment:
could you please use one style of naming variables?

Reply:
We definetely going to use the same naming